tools/Cron_jobs/mailwatch_quarantine_maint.php

```diff
@@ -66,16 +66,26 @@ function quarantine_reconcile()
 
     function quarantine_clean()
     {
-        $oldest = date('U', strtotime('-' . QUARANTINE_DAYS_TO_KEEP . ' days'));
+        $oldestspam = date('U', strtotime('-' . QUARANTINE_DAYS_TO_KEEP . ' days'));
+        $oldestnonspam = date('U', strtotime('-' . QUARANTINE_DAYS_TO_KEEP_NONSPAM . ' days'));
```

Comment:
There should be a check here to confirm if constant exists, or fallback on `QUARANTINE_DAYS_TO_KEEP`